src/clocksi_interactive_tx_coord_sup.erl

```diff
@@ -34,8 +34,8 @@ start_link() ->
     supervisor:start_link({local, ?MODULE}, ?MODULE, []).
 
 start_fsm(Args) ->
-    _Res = random:seed(dc_utilities:now()),
-    Random = random:uniform(?NUM_SUP),
+    _Res = rand:seed(exsplus, erlang:timestamp()),
```

Comment:
This is a backwards incompatible change, right?  For instance, will this compile with Erlang 18?  (That's the most commonly deployed Erlang.)